libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -963,6 +994,81 @@ def Delete(
             self.FormatLogMessage(
                 'Could not find disk: {0:s}, skipping'.format(disk_name)))
 
+  def AssignExternalIp(self, net_if: str, ip_addr: Optional[str]) -> None:
+    """Assigns an external IP to an instance's network interface.
+
+    The instance must not have an IP assigned to the network interface when
+    calling this method. If the IP address is specified, it must be one that
+    is available to the project.
+
+    Args:
+      net_if (str): The instance's network interface to which the IP address
+        must be assigned.
+      ip_addr (Optional[str]): The static IP address that exposes the network
```

Comment:
In the docstring itself, you can write optional parameters as `param (str): Optional. [description]`